src/main/java/com/github/arkadiusz97/sms_filter_service/smsfilterservice/sms_operator_service/controller/SmsController.java

```diff
@@ -0,0 +1,21 @@
+package com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.controller;
+
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.Sms;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.SmsServiceResponse;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.service.SmsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class SmsController {
+
+    @Autowired
+    SmsService smsService;
+
+    @PostMapping("smsOperatorService/handleSms")

```

Comment:
Zwróć uwagę na nazwę endpoint'u - jest tu kilka rzeczy, które fajnie byłoby zmienić. Dam Ci wskazówkę, że chodzi o sposób zapisu oraz o wyraz `handle` - wiesz dlaczego chciałbym to zmienić?

Reply:
Nazwa endpointu powinna odnosić się do jakiejś rzeczy a nie do akcji, czyli słowa handle. Poza tym, słowa możnaby rozdzielić myślnikiem. Poprawiona nazwa endpointa może wyglądać tak: "sms-operator-service/sms"